src/js/components/devices/device-details/authsets/authsetlistitem.js

```diff
@@ -99,18 +131,36 @@ const AuthsetListItem = ({ authset, confirm, device, isExpanded, limitMaxed, loa
     return onConfirm(DEVICE_STATES.accepted);
   };
 
+  const onCopied = (_, result) => {
+    setCopied(result);
+    setTimeout(() => setCopied(false), 5000);
+  };
+
   let key = <a onClick={onShowKey}>show key</a>;
   let content = [
     padder,
     <p className="bold expanded" key="content">
       {loading === authset.id ? 'Updating status' : `${confirmMessage} Are you sure you want to continue?`}
     </p>
   ];
+
   if (showKey) {
     content = [
-      <code className="expanded pre-line" key="content">
-        {authset.pubkey}
-      </code>,
+      <div key="content">
+        <CopyToClipboard text={authset.pubkey} onCopy={onCopied}>
+          <IconButton style={{ float: 'right', margin: '-20px 0 0 10px' }}>
+            <CopyPasteIcon />
+          </IconButton>
+        </CopyToClipboard>
+        <code className="pre-line">{authset.pubkey}</code>
+        {copied && <p className="green fadeIn">Copied key to clipboard.</p>}
+        <Divider style={{ marginTop: theme.spacing(), marginBottom: theme.spacing() }} />
+        <div>
+          SHA256 Checksum
```

Comment:
Would call it "Checksum (SHA256)" to align with label in releases (https://hosted.mender.io/ui/#/releases), we just call it Checksum there in fact.

Reply:
Changed it to just "Checksum" to stay in line with releases, but once you hover over it, the algorithm is shown:
<img width="566" alt="Screenshot 2021-07-12 at 12 07 54" src="https://user-images.githubusercontent.com/482243/125270697-372ff500-e30a-11eb-84f3-3c259e7169b9.png">

Unfortunately adding this to the releases list is a bit more involved, otherwise I'd have added it here too...